module/Entra/AdditionalFunctions/Get-EntraApplicationTemplate.ps1

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function Get-EntraApplicationTemplate {
+    [CmdletBinding(DefaultParameterSetName = 'GetQuery')]
+    param (
+    [Parameter(ParameterSetName = "GetById", Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Id
```

Comment:
We could support more parameters

-Property
We can select specific properties
![image](https://github.com/user-attachments/assets/d72f7532-14f3-42e7-a92a-c2f1c325c838)

-Top
![image](https://github.com/user-attachments/assets/e2675511-a7a8-4bb4-8cc2-4888d53f47a8)

Reply:
added the paramters